app/Observers/UserObserver.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace App\Observers;
+
+use App\User;
+use Illuminate\Support\Facades\Hash;
+
+class UserObserver {
+
+  public function created(User $user) {
```

Comment:
I can make the switch, when I get home tonight. Two birds getting stoned at once is better than one bird getting hit with a stone